src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ContentDbContext.cs

```diff
@@ -147,6 +148,16 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<Release>()
                 .HasQueryFilter(r => !r.SoftDeleted);
 
+            modelBuilder.Entity<ReleaseStatus>()
```

Comment:
Did this config make any difference for the config that's already in code? Perhaps it altered the cascade action, but the `.IsRequired(false)` will already be picked up as the field is marked as optional in ReleaseStatus.cs.

Reply:
I've made the change and rebased it on to the commit where all the other ContentDbContext changes are, so nobody looks at the PR changes commit and wonders why it was removed